src/common/options.cc

```diff
@@ -1986,7 +1986,7 @@ std::vector<Option> get_global_options() {
     .set_description(""),
 
     Option("osd_skip_data_digest", Option::TYPE_BOOL, Option::LEVEL_DEV)
-    .set_default(true)
+    .set_default(false)
```

Comment:
Based off the comment, can we set this off? Or did it just get added to the luminous branch after the release?

Reply:
The feature was newly (and incompletely) backported in 12.2.6, so setting it off reverts to earlier luminous behavior.  The release notes ask users to set this (or osd_distrust_data_digest) to true if they installed 12.2.6 to work around the problem.  See https://github.com/ceph/ceph/pull/23057